modules/ROOT/pages/managing-instances/vector-optimization.adoc

```diff
@@ -0,0 +1,26 @@
+= Vector optimization
+
+label:AuraDB-Professional[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+== Overview
+
+Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations. 
+It is available for AuraDB instances with more than 4GB of memory.
+
+Enable vector optimization using a toggle during instance setup. 
+This configuration may increase memory use, which could impact performance for other queries.
+
+You can enable vector optimization during instance creation for instances of 4GB RAM or higher and only for Neo4j 5.
+Vector optimization is available across all supported providers and regions.
+
+The toggle to enable vector optimization is available under *Additional settings*.
+You can view the current vector configuration status in *Instance Details*.
+
+If you lower the instance size below 4GB, vector optimization is disabled automatically.
+
+If you clone your instance to a new instance, the new instance inherits the vector optimization settings of the original instance.
+But if you clone to an existing instance, it *does not* inherit the original instance's vector optimization settings.
+
+To learn more about how to use vector indexes, see link:https://neo4j.com/docs/cypher-manual/current/indexes/semantic-indexes/vector-indexes/[Cypher Manual -> Vector indexes]
```

Comment:
```suggestion
== Vector optimization

label:AuraDB-Professional[]
label:AuraDB-Business-Critical[]
label:AuraDB-Virtual-Dedicated-Cloud[]


Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations.
It is available for AuraDB instances with more than 4GB of memory across all supported providers and regions.

This configuration may impact memory use and thus performance for other queries.

You can enable vector optimization with a toggle under *Additional settings* in the instance details.
This can be done both during instance creation and for existing instances.
You can view the current vector configuration status in *instance details*.

If you lower the instance size below 4GB, vector optimization is disabled automatically.

If you clone your instance to a new instance, the new instance inherits the vector optimization settings of the original instance.
But if you clone to an existing instance, the existing instance *does not* inherit the original instance's vector optimization settings.

To learn more about how to use vector indexes, see link:https://neo4j.com/docs/cypher-manual/current/indexes/semantic-indexes/vector-indexes/[Cypher Manual -> Vector indexes]
```

Reply:
Then cut and paste this section into managing-instances/instance-details.adoc on line 57.